src/components/SnartSluttPaSykepengene/MaxDateIngress.tsx

```diff
@@ -70,11 +70,7 @@ function MaxDateContent(): ReactElement {
 }
 
 function MaxDateIngress(): ReactElement {
-  return (
-    <Box>
-      <MaxDateContent />
-    </Box>
-  )
+  return <MaxDateContent />
```

Comment:
hm, kanskje like greit å eksportere maxdatecontent? Og evnt kalle den for maxdateingress.